auth-web/devtotestpipeline

```diff
@@ -0,0 +1,11 @@
+kind: "BuildConfig"
+apiVersion: "v1"
+metadata:
+  name: "deploy-ui-to-test-pipeline"
+spec:
+  source:
+    git:
+      uri: "https://github.com/saravankumarpa/sbc-auth"
```

Comment:
Is this file just for your own testing? Do we want to commit it?

Reply:
Removed it. It was a helper file.Need not to be part of code base. May be we can create a gist for it.